ts/parts/StackingAxis.ts

```diff
@@ -166,13 +168,18 @@ class StackingAxisAdditions {
         const chart = axis.chart;
         const renderer = chart.renderer;
         const stacks = stacking.stacks;
+        const stackLabelsAnim = axis.options.stackLabels.animation;
```

Comment:
Maybe if we expand the `getDeferTime` function to take the actual animation options into account, we can make this function so general that both data labels and stack labels can use it? For data labels it would only look at that particular series - for stack labels it would look at all series. Then it would return `{ duration, defer }` that can be applied directly to the `animate` function.